charts/rs-server-station-secrets/values.yaml

```diff
@@ -32,6 +32,21 @@ app:
         client_id: client_id
         client_secret: client_secret
         authorization: Basic test
+    adgs2:
```

Comment:
@Padeanu I've added this to add the env vars RSPY__TOKEN__xxx for adgs2. We'll need to redeploy this also.